### DISCUSSION TITLE
Remapping `<leader><space>` to Find Files (cwd) doesn't work.

### DISCUSSION THREAD

**Discussion by rasibn** [LazyVim/LazyVim#3978] [Q&A] [answered]
I was following the instruction on https://www.lazyvim.org/configuration/plugins and trying to map `<leader><space>` to `Telescope find_files`, but I am getting some sort of error that makes this keymap not work. You can see that even the command description became empty somehow. Here is my `telescope.lua` file in the `plugins` directory:

### Code
 ```lua
return {
  "nvim-telescope/telescope.nvim",
  keys = { "<leader><space>", "<cmd>Telescope find_files<cr>", desc = "Find Files (test)" },
}
```

### Screenshot
![image](https://github.com/LazyVim/LazyVim/assets/49709116/95214b19-32a8-4ece-9729-457494125b84)

**Comment by dpetka2001** [comment, marked as answer]
One more pair of curly braces in `keys`, so it should be like `keys = { { "keymap you want here"}} `.
